js1/dom-innerhtml/innerhtml.md

```diff
@@ -1,6 +1,6 @@
 ## Vlastnost innerHTML
 
-Z předchozí části už umíme změnit obsah nějakého DOM elementu pomocí vlastnosti `textContent`. Takto však můžeme pracovat pouze s čistě textovým obsahem. Co kdybychom však do nějakého elementu chtěli vložit kus HTML? K tomu nám poslouží vlastnost `innerHTML`. Tuto vlastnost jste potkáváli již od samého počátku kurzu při používání :var[document.body.innerHTML]. Vlastnost `innerHTML` však kromě elementu `body` najdeme i na většině ostatních elementů.
+Z předchozí části už umíme změnit obsah nějakého DOM elementu pomocí vlastnosti `textContent`. Takto však můžeme pracovat pouze s čistě textovým obsahem. Co kdybychom však do nějakého elementu chtěli vložit kus HTML? K tomu nám poslouží vlastnost `innerHTML`. Tuto vlastnost jste potkávali již od samého počátku kurzu při používání `document.body.innerHTML`. Vlastnost `innerHTML` však kromě elementu `body` najdeme i na většině ostatních elementů.
```

Comment:
Za mě super. Moc totiž nevím, co se tím `:var[]` v kodim-cms sleduje kromě toho, že se to hůře píše a méně nástrojů tomu rozumí.